testing/testenv/testlib/README.md

```diff
@@ -30,8 +30,22 @@ ProjData generator
     e.g. TSW, capacity 100:
         "TSW 100 *" .. "TSW 001 *"
 
+    dumps data to file
+
     additional features
-    -   set signal type for signal data
+    -   set SIG type for signal data
+    -   set LCR type for level crossing data
+```
+
+**MutableArray.h**
+```
+array types for test issues
```

Comment:
"array types for test issues"

then it should be called TestArrays.h

Reply:
OK, done